src/tracks/Pattern.cpp

```diff
@@ -481,11 +481,19 @@ void Pattern::addSteps()
 void Pattern::cloneSteps()
 {
 	int oldLength = m_steps;
+	int newTacts = (oldLength / MidiTime::stepsPerTact()) + 1;
+	int startStep = 0; //The cloning process will normally start from 0
+	if ((newTacts%2==0)&&(newTacts>3)) {
```

Comment:
Infix operators (=, +, -, *, /, etc.) should have a space before and after by the coding conventions. Please fix this. 
https://github.com/LMMS/lmms/wiki/Coding-conventions